### PR TITLE
add patching info to images doc

### PR DESCRIPTION
**Pull request by rosemarymarano**
Add "Apply patched images to a running domain" section to (old) Base Images doc, now WebLogic Server images, located in the TOC between Manager Operators and Manage WebLogic Domains.

### CODE REVIEW

File: documentation/staging/content/userguide/base-images/_index.md [hunk 1]
```diff
@@ -131,24 +132,87 @@ to build the image and apply the patches.
...
+##### Domain in PV
+
+Edit the Domain Resource image reference with the new image name/tag (for example, `oracle/weblogic:12.2.1.4-patched`).
+Then, the operator performs a rolling restart of the WebLogic domain to update the Oracle Home of the servers.
+For information on server restarts, see [Restarting]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting.md" >}}).
+
+##### Model in Image
+Use the WIT [`rebase`](https://github.com/oracle/weblogic-image-tool/blob/master/site/rebase-image.md) command
+to update the Oracle Home for an existing image with the model and archive files in the image using the patched Oracle Home from a
+patched container image. Then, the operator performs a rolling update of the domain, updating the Oracle Home of each server pod.
```
Comment (tbarnes-us) [inline documentation/staging/content/userguide/base-images/_index.md:189]: Consider linking to the life-cycle documentation for rolling update at each mention of rolling update.

Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md:189]: Will do.

File: documentation/staging/content/userguide/managing-domains/domain-in-image/_index.md [hunk 2]
```diff
@@ -2,15 +2,14 @@
...
-{{% notice warning %}}
-Oracle strongly recommends storing a domain image as private in the registry.
-A container image that contains a WebLogic domain home has sensitive information
-including keys and credentials that are used to access external resources
-(for example, the data source password). For more information, see
-[WebLogic domain in container image protection]({{<relref "/security/domain-security/image-protection#weblogic-domain-in-container-image-protection">}}).
+
+{{% notice note %}}
+Oracle strongly recommends that images containing WebLogic domains
+be kept in a private repository.
```
Comment (tbarnes-us) [inline documentation/staging/content/userguide/managing-domains/domain-in-image/_index.md:14]: I think it's useful to include the reasoning behind the recommendation, and fine to duplicate wording of a similar warning that this pull has elsewhere.

File: documentation/staging/content/userguide/base-images/_index.md (hunk 1 above)
Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md:189]: There is so much detailed information about rolling restarts, I'm tempted to cross-reference the Overview section, where it all begins, https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/restarting/#overview
Do you have a better idea?

File: documentation/staging/content/userguide/managing-domains/domain-in-image/_index.md (hunk 2 above)
Comment (rosemarymarano) [reply documentation/staging/content/userguide/managing-domains/domain-in-image/_index.md:14]: The doc you are looking at is draft = true; the one we removed. I did duplicate this note in the current doc, with the larger explanation.

File: documentation/staging/content/userguide/base-images/_index.md (hunk 1 above)
Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md:189]: Actually, there is this cross-reference also, which might be a better one for rolling restarts, http://localhost:1313/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#rolling-restarts. I think I will use it.

Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md:189]: I think #restarting-servers is a better link than #rolling-restarts (restarting servers discusses what happens when you change the image, rolling-restarts goes into detail about how restarts are handled at the cluster level).

File: documentation/staging/content/userguide/base-images/_index.md [hunk 3]
```diff
@@ -131,24 +132,87 @@ to build the image and apply the patches.
...
+
+#### Apply patched images to a running domain
+
+When updating the WebLogic binaries of a running domain in Kubernetes with a patched container image,
+the operator applies the update in a Zero Downtime (ZDT) fashion. The procedure for the operator
+to update the running domain differs depending on the [domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}). A
+difference between the domain home source types is the location of the domain home:
+
+* Domain in PV: The container image contains the JDK and WebLogic Server binaries. The domain home is located in a Persistent Volume (PV).
+* Model in Image: The container image contains the JDK, WebLogic Server binaries, and a WebLogic Deployment Tooling (WDT) model file and application archive file.
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md:167]: Got it.

File: documentation/staging/content/userguide/base-images/_index.md (hunk 1 above)
Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md:189]: Agree. I'll change them back.

File: documentation/staging/content/userguide/base-images/_index.md [hunk 4]
```diff
@@ -1,19 +1,21 @@
 ---
```
Comment (tbarnes-us) [inline documentation/staging/content/userguide/base-images/_index.md:1]: I have some general questions for @ddsharpe  about updating the WDT install - the directions in this chapter only cover updating JRE & WLS - but not WDT. If there's no time pressure, I recommend fixing that. IMO, the chapter should specifically discuss how to download WDT, put it in teh WIT cache, and tell WIT to include the updated WDT install in the rebase.  Is there time pressure @mriccell ?

Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md:1]: And what are the incantations for doing this @ddsharpe ?

**Review by rjeberhard — APPROVED** [review]

**Comment by tbarnes-us** [comment]
General comment: I just learned that 'rebase' might be new to WDT - it seems to arrive in 1.9.11?  If so, then I assume the doc should mention that we need 1.9.11 or later to make it work?  @mriccell FYI

**Review by tbarnes-us — APPROVED** [review]
Since we want to get this out very soon, we can defer my questions for @ddsharpe -

**Pull request merged**